### ISSUE TITLE
Running ./tools/update does not auto shutdown existing replicator

### ISSUE THREAD

**Issue by GoogleCodeExporter** [epermana/tungsten-replicator#599]
```
What steps will reproduce the problem?

1. Download a recent replicator build (I've tried 250 and 255)
2. Try running ./tools/update 
3. Update fails with a failure to restart the replicator, and the old 
replicator is still running

What is the expected output?

Should have an updated replicator config and:

Deployment finished

What do you see instead?

DEBUG >> host1 >> Execute 
`/opt/continuent/tungsten/tungsten-replicator/bin/replicator status`
DEBUG >> host1 >> Execute 
`/opt/continuent/tungsten/tungsten-replicator/bin/replicator start`
ERROR >> host1 >> Failed: 
/opt/continuent/tungsten/tungsten-replicator/bin/replicator start, RC: 256, 
Result: Starting Tungsten Replicator Service...
Waiting for Tungsten Replicator Service......
WARNING: Tungsten Replicator Service may have failed to start.:
./ruby/configurator.rb:1223:in `cmd_result'
./ruby/configure/deployment_steps/deployment_step_services.rb:32:in 
`apply_config_services'
./ruby/configure/deployment_steps/deployment_step_services.rb:28:in `each'
./ruby/configure/deployment_steps/deployment_step_services.rb:28:in 
`apply_config_services'
./ruby/configure/deployment_steps/configure_deployment_core.rb:41:in `send'
./ruby/configure/deployment_steps/configure_deployment_core.rb:41:in `deploy'
./ruby/configure/deployment_steps/configure_deployment_core.rb:38:in `each'
./ruby/configure/deployment_steps/configure_deployment_core.rb:38:in `deploy'
./ruby/configure/configure_deployment.rb:58:in `deploy_config'
./ruby/configure/configure_deployment_handler.rb:64:in `deploy_config'
./ruby/configure/configure_deployment_handler.rb:50:in `deploy'
./ruby/configure/configure_deployment_handler.rb:47:in `each'
./ruby/configure/configure_deployment_handler.rb:47:in `deploy'
./ruby/configure/configure_deployment.rb:43:in `deploy'
./ruby/configurator.rb:281:in `run'
ruby/configure.rb:51
DEBUG >> host1 >> Is 'host1' the current host?

What is the possible cause?

Existing replicator is not being shutdown before starting the new one

What is the proposed solution?

Run the replicator stop before starting the new one.

Additional information

Two logs attached, one shows what happens doing an update without shutting down 
first, the other with a shutdown (-shutdown). 

```

Original issue reported on code.google.com by `mc.br...@continuent.com` on 22 May 2013 at 3:05
- Merged into: #597

Attachments:
- [tr-upgrade.log.bz2](https://storage.googleapis.com/google-code-attachments/tungsten-replicator/issue-599/comment-0/tr-upgrade.log.bz2)
- [tr-upgrade-shutdown.log.bz2](https://storage.googleapis.com/google-code-attachments/tungsten-replicator/issue-599/comment-0/tr-upgrade-shutdown.log.bz2)
